data_conversion/index.html

```diff
@@ -30,6 +30,7 @@
     <a href="#sfr">Star Formation Rates</a>
     <a href="#bh">Black Holes</a>
     <a href="#smhmr">Stellar Mass-Halo Mass Relation</a>
+    <a href="#sfr_halo">Star Formation Rates-Halo Mass Relation</a>
```

Comment:
No need for plural in star formation rate(s) when putting it as the first one, the plural is referring to the number of plots. Perhaps we should have changed it to `Stellar Mass-Halo Mass Relations`...